app/default/application_routes.py

```diff
@@ -346,20 +348,35 @@ def submit_application():
     )
     submitted = format_payload_and_submit_application(application_id)
 
-    application_id = submitted.get("id")
-    application_reference = submitted.get("reference")
-    application_email = submitted.get("email")
     with force_locale(application.language):
-        return render_template(
-            "application_submitted.html",
-            application_id=application_id,
-            application_reference=application_reference,
-            application_email=application_email,
-            fund_name=fund_data.name,
-            fund_short_name=fund_data.short_name,
-            round_short_name=round_data.short_name,
-            migration_banner_enabled=Config.MIGRATION_BANNER_ENABLED,
-        )
+        if fund_data.short_name in ("COF-EOI",):
+            eoi_results = get_application_eoi_response(application)
+            return render_template(
+                "eoi_submitted.html",
+                eoi_pass=eoi_results["decision"] in [Eoi_Decision.PASS, Eoi_Decision.PASS_WITH_CAVEATS],
+                fund_name=fund_data.name,
+                round_name=round_data.title,
+                fund_short_name=fund_data.short_name,
+                round_short_name=round_data.short_name,
+                round_prospectus=round_data.prospectus,
+                migration_banner_enabled=Config.MIGRATION_BANNER_ENABLED,
+            )
```

Comment:
I'm guessing in the future we'll have some way to mark a fund as EOI, instead of having to check the short name exists inside a tuple/list?

Reply:
just want to make the code generic if we have EOI for other rounds(e.g `NSTF-EOI`) then in that case we just append it to the tuple.